selfdrive/car/toyota/carstate.py

```diff
@@ -64,7 +64,7 @@ def update(self, cp, cp_cam):
 
     if self.accurate_steer_angle_seen:
       # Offset seems to be invalid for large steering angles
-      if abs(ret.steeringAngleDeg) < 90:
+      if abs(ret.steeringAngleDeg) < 90 and cp.can_valid:
```

Comment:
Maybe we can move this to the check on line 62? `if abs(torque_sensor_angle_deg) > 1e-3:` The next check relies on the update call

Reply:
This is fine, right? The next lines already check for initialized.

Reply:
Yeah, this is more clear now. It would just be a very small optimization